addon/components/power-select.js

```diff
@@ -86,6 +86,7 @@ export default Component.extend({
   closeOnSelect: fallbackIfUndefined(true),
   defaultHighlighted: fallbackIfUndefined(defaultHighlighted),
   typeAheadMatcher: fallbackIfUndefined(defaultTypeAheadMatcher),
+  highlightOnHoverEnabled: fallbackIfUndefined(true),
```

Comment:
Can you just rename this to `highlightOnHover` (without the `enabled` suffix). Seems redundant.